Error while installing most recent version (with tooltips fix)

@stormpython thanks for the quick fix. I am still getting the following error when trying to install the plugin. Not sure if this issue is related to the previous one.

```
./kibana plugin -i heatmap -u https://github.com/stormpython/heatmap/archive/master.zip
Installing heatmap
Attempting to transfer from https://github.com/stormpython/heatmap/archive/master.zip
Transferring 1811145 bytes....................
Transfer complete
Extracting plugin archive
Extraction complete
Optimizing and caching browser bundles...
Plugin installation was unsuccessful due to error "Optimizations failure.

    WARNING in ../src/ui/public/vis/schemas.js
    There is another module with an equal name when case is ignored.
    This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
    Rename module if multiple modules are expected or use equal casing if one module is expected.

    WARNING in ../src/ui/public/Vis/Schemas.js
    There is another module with an equal name when case is ignored.
    This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.
    Rename module if multiple modules are expected or use equal casing if one module is expected.

    ERROR in ../installedPlugins/heatmap/public/heatmap.js
    Module not found: Error: Cannot resolve 'file' or 'directory' /Users/vineet/kibana-4.5.1-darwin-x64/src/ui/public/template_vis_type/template_vis_type in /Users/vineet/kibana-4.5.1-darwin-x64/installedPlugins/heatmap/public
     @ ../installedPlugins/heatmap/public/heatmap.js 11:32-81
```

--- a/public/heatmap.js
+++ b/public/heatmap.js
@@ -1,13 +1,14 @@
 require('plugins/heatmap/heatmap.less');
-require('plugins/heatmap/heatmap_tooltip.css');
+require('plugins/heatmap/heatmap_tooltip.less');
 require('plugins/heatmap/color_directive.js');
 require('plugins/heatmap/lib/heatmap_controller.js');
 require('plugins/heatmap/lib/heatmap_directive.js');
 require('plugins/heatmap/heatmap_tooltip_directive.js');
 
+
 function HeatmapProvider(Private) {
-  var TemplateVisType = Private(require('ui/template_vis_type/template_vis_type'));
-  var Schemas = Private(require('ui/vis/schemas'));
+  var TemplateVisType = Private(require('ui/template_vis_type/TemplateVisType'));
+  var Schemas = Private(require('ui/Vis/Schemas'));
   var colors = require('plugins/heatmap/colors.js');
 
   return new TemplateVisType({

--- a/public/heatmap_tooltip.less
+++ b/public/heatmap_tooltip.less
@@ -1,15 +1,19 @@
+@import (reference) "~ui/styles/variables";
+
 .heatmap-tooltip {
   position: absolute;
   width: auto;
-  background-color: #222222;
-  -webkit-border-radius: 10px;
-  -moz-border-radius: 10px;
-  border-radius: 10px;
+  padding: 5px !important;
+  background: fadeout(@gray-darker, 7%);
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
   -webkit-box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.4);
-  -mox-box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.4);
+  -moz-box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.4);
   box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.4);
   pointer-events: none;
-  white-space: nowrap;
+  white-space: nowrap;  
+  color: #ecf0f1 !important;   
 }
 
 .heatmap-tooltip-list {

